tests/test_models.py

```diff
@@ -338,7 +334,6 @@ def test_mars_tms():
         MARS_MERCATOR,
         extent_crs=MARS2000_SPHERE,
         title="Web Mercator Mars",
-        geographic_crs=MARS2000_SPHERE,
```

Comment:
maybe add a test here to ensure the new `geographic_crs` is appropriate?

Reply:
yeah, I'm not sure how then

Reply:
`mars_tms.crs._pyproj_crs.geodetic_crs == MARS2000_SPHERE` should be fine

Reply:
`mars_tms.crs._pyproj_crs.geodetic_crs` isn't equal to `MARS2000_SPHERE` but I've added tests showing the the transformation from using `.crs._pyproj_crs.geodetic_crs` and `MARS2000_SPHERE` give the same results

Reply:
looks equal to me? what am I missing?

```
In [7]: mars_tms.crs._pyproj_crs.geodetic_crs.to_wkt()
Out[7]: 'GEOGCRS["unknown",DATUM["unknown",ELLIPSOID["unknown",3396190,0,LENGTHUNIT["metre",1,ID["EPSG",9001]]]],PRIMEM["Reference meridian",0,ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],CS[ellipsoidal,2],AXIS["longitude",east,ORDER[1],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],AXIS["latitude",north,ORDER[2],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]]]'

In [8]: MARS2000_SPHERE.to_wkt()
Out[8]: 'GEOGCRS["unknown",DATUM["unknown",ELLIPSOID["unknown",3396190,0,LENGTHUNIT["metre",1,ID["EPSG",9001]]]],PRIMEM["Reference meridian",0,ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],CS[ellipsoidal,2],AXIS["longitude",east,ORDER[1],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],AXIS["latitude",north,ORDER[2],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]]]'

```

Reply:
🤔 I though I tested `mars_tms.crs._pyproj_crs.geodetic_crs == MARS2000_SPHERE` but it was returning false, let me re-check